src/logs/metadata_engine/metadata_engine.py

```diff
@@ -139,14 +142,27 @@ def _apply_rule(rule, record, parsed_record):
     if rule.aws_loggroup_pattern and "log_group" in record:
         extracted_values = parse_aws_loggroup_with_grok_pattern(record["log_group"], rule.aws_loggroup_pattern)
         record.update(extracted_values)
+    if rule.log_content_parse_type == "json":
+        try:
+            record["log_content_parsed"] = json.loads(parsed_record.get("content", {}))
+        except Exception as ex:
+            logging.log_error_with_stacktrace(ex, f"Encountered exception when parsing log content as json, requested by rule for {rule.entity_type_name}")
+    else:
+        record["log_content_parsed"] = parsed_record.get("content", "")
```

Comment:
Maybe then "content" should be "content" and not "log_content_parsed"?

Reply:
why not, now that I have the else for lambda. Will call it "log_content", though.